src/peers/gov/nasa/jpf/vm/JPF_java_lang_System.java

```diff
@@ -83,10 +83,15 @@ int createPrintStream (MJIEnv env, int clsObjRef){
   public int createSystemOut____Ljava_io_PrintStream_2 (MJIEnv env, int clsObjRef){
     return createPrintStream(env,clsObjRef);
   }
-  
+
   @MJI
-  public int createSystemErr____Ljava_io_PrintStream_2 (MJIEnv env, int clsObjRef){
-    return createPrintStream(env,clsObjRef);
+  public int createSystemErr____Ljava_io_PrintStream_2(MJIEnv env, int clsObjRef) {
+    return createPrintStream(env, clsObjRef);
+  }
+
+  @MJI
+  public int lineSeparator____Ljava_lang_String_2(MJIEnv env, int objRef) {
+    return env.newString(System.lineSeparator());
```

Comment:
The peer maps the line Separator to the line Separator of the host JVM in this line of code.

Reply:
Thanks, in that case the PR is good to go!